blockchain/chain_sync/src/tipset_syncer.rs

```diff
@@ -1258,7 +1258,7 @@ async fn validate_block<
         ));
     } else if header.timestamp() > time_now {
         warn!(
-            "Got block from the future, but within clock drift threshold, {} > {}",
+            "Got block from the future, but within fvm_shared::clock drift threshold, {} > {}",
```

Comment:
I don't think this is an unnecessary implementation detail, let's keep it as it was

Reply:
Fixed